Library/Homebrew/completions/fish.erb

```diff
@@ -104,22 +104,14 @@ function __fish_brew_ruby_parse_json -a file parser -d 'Parses given JSON file w
 end
 
 function __fish_brew_suggest_formulae_all -d 'Lists all available formulae with their descriptions'
-    # store the brew cache path in a var (because calling (brew --cache) is slow)
-    set -q __brew_cache_path
-    or set -gx __brew_cache_path (brew --cache)
-
-    if test -f "$__brew_cache_path/descriptions.json"
-        __fish_brew_ruby_parse_json "$__brew_cache_path/descriptions.json" \
-            '.each{ |k, v| puts([k, v].reject(&:nil?).join("\t")) }'
-    else
-        brew formulae
-    end
+    brew formulae
 end
 
 function __fish_brew_suggest_formulae_installed
-    brew list --formula
+    command ls -1 (brew --cellar)
 end
 
+
 function __fish_brew_suggest_formulae_outdated -d "List of outdated formulae with the information about potential upgrade"
     brew outdated --formula --verbose \
```

Comment:
This is very slow and I wonder if there's value added by the completions. 

```fish
❯ hyperfine 'brew outdated --formula --verbose' --shell fish
Benchmark 1: brew outdated --formula --verbose
  Time (mean ± σ):      1.199 s ±  0.052 s    [User: 0.892 s, System: 0.182 s]
  Range (min … max):    1.153 s …  1.301 s    10 runs
```

Reply:
If someone added it: I guess they consider it to add value. 1s doesn't seem excessive?